src/java.base/share/man/java.1

```diff
@@ -2974,11 +2974,18 @@ different JDK version.
 .RE
 .RE
 .TP
-.B \f[CB]\-XX:+ExtendedDTraceProbes\f[R]
-\f[B]Linux and macOS:\f[R] Enables additional \f[CB]dtrace\f[R] tool probes
-that affect the performance.
-By default, this option is disabled and \f[CB]dtrace\f[R] performs only
-standard probes.
+.B \f[CB]\-XX:+DTraceAllocProbes\f[R]
```

Comment:
The three newly documented flags should all be marked "Linux and macOS". It is somewhat of a poor design that the flags are available on all platforms but only have an effect on systems with DTrace or SystemTap support - which (for our main platforms) is Linux and macOS.

Reply:
done